### PR TITLE
Rodrigom/finetuning embedding

### PR DESCRIPTION
**Pull request by snova-rodrigom**
- Added support to auth issues workaround
- Adapted job progress logic
- Adapted list checkpoints code
- Adapted promote checkpoint code

### CODE REVIEW

File: utils/fine_tuning/config.yaml
Comment (snova-jorgep) [inline utils/fine_tuning/config.yaml:1]: can we let this file with previous llms config?

Comment (snova-jorgep) [reply utils/fine_tuning/config.yaml:1]: given now we have a config one for embeddings with custom config?

File: utils/fine_tuning/src/snsdk_wrapper.py [hunk 1]
```diff
@@ -342,6 +347,7 @@ def create_project(
                 project_name=project_name, description=project_description
             )
             if create_project_response["status_code"] == 200:
+                project_id = create_project_response["id"]
                 project_id = create_project_response["id"]
```
Comment (snova-jorgep) [inline utils/fine_tuning/src/snsdk_wrapper.py:351]: this line is duplicated

**Review by snova-jorgep — CHANGES_REQUESTED** [review]

File: utils/fine_tuning/config.yaml
Comment (snova-rodrigom) [reply utils/fine_tuning/config.yaml:1]: sure thing, I've reverted that config to the original state

**Review by snova-rodrigom — COMMENTED** [review]
commented on fixes coming from @snova-jorgep

**Review by snova-jorgep — APPROVED** [review]

**Pull request merged**